### PR TITLE
Added quote from Melina Marchetta

### PR DESCRIPTION
**Pull request by Kryze**
## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [x] The quote(s) added are about **learning a foreign language**.
- [x] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -335,6 +335,10 @@
     "author": "Mark Amidon",
     "sentence": "Language is the means of getting an idea from my brain into yours without surgery."
   },
+  {
+    "author":"Melina Marchetta",
```
Comment (Kvaibhav01) [inline server/data/quotes.json:339]: Please add a space between the key and its value as shown:

```suggestion
    "author": "Melina Marchetta",
```

File: server/data/quotes.json [hunk 2]
```diff
@@ -335,6 +335,10 @@
...
+    "sentence":"Because without our language, we have lost ourselves. Who are we without our words?"
```
Comment (Kvaibhav01) [inline server/data/quotes.json:340]: The same goes for this.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Thanks for the PR, just make the necessary changes as shown.

**Review by jayehernandez — CHANGES_REQUESTED** [review]
Same comments as @Kvaibhav01! Please request for review after making the necessary changes!

**Comment by jayehernandez** [comment]
Closing this stale PR :)

**Pull request closed without merging**